paper/paper.Rmd

```diff
@@ -42,23 +49,113 @@ link-citations: yes
+
+`{datawizard}` provides various functions for cleaning and preparing data (see Table 1).
+
+Function           | Operation                             |
+------------------ | --------------------------------------|
+`data_filter()`    | to select only certain *observations* |
+`data_select()`    | to select only a few *attributes*     |
+`data_extract()`   | to extract a single *attribute*       |
+`data_rename()`    | to rename attributes                  |
+`reshape_longer()` | to convert data from wide to long     |
+`reshape_wider()`  | to convert data from long to wide     |
+`data_join()`      | to join two data frames               |
+    ...            |        ...                            |
+
+Table: The table below lists a few key functions offered by *datawizard* for data wrangling. To see the full list, see the package website: <https://easystats.github.io/datawizard/>
+
+We will look at one example function that converts data in wide format to tidy/long format:
+
+```{r}
+stocks <- data.frame(
+  time = as.Date('2009-01-01') + 0:4,
+  X = rnorm(5, 0, 1),
+  Y = rnorm(5, 0, 2)
+)
+
+stocks
+
+data_to_long(
+  stocks,
+  select = -c("time"),
+  colnames_to = "stock",
+  values_to = "price"
+)
+```
+
+## Data transformations
+
+Even after getting the raw data in the needed format, we may further need to transform certain variables further to meet requirements imposed by the statistical model.
+
+`{datawizard}` provides a rich collection of such functions for transforming variables (see Table 2).
+
+Function           | Operation                                     |
+------------------ | ----------------------------------------------|
+`standardize()`    | to center and scale data                      |
+`normalize()`      | to scale variables to 0-1 range               |
+`adjust()`         | to adjust data for effect of other variables  |
+`data_shift()`     | to shift numeric value range                  |
+`ranktransform()`  | to convert numeric values to integer ranks    |
+    ...            |        ...                                    |
+
+Table: The table below lists a few key functions offered by *datawizard* for data transformations. To see the full list, see the package website: <https://easystats.github.io/datawizard/>
+
+We will look at one example function that standardizes (i.e. centers and scales) data so that it can be expressed in terms of standard deviation:
+
+```{r}
+d <- data.frame(
+  a = c(-2, -1, 0, 1, 2),
+  b = c(3, 4, 5, 6, 7)
+)
+
+standardize(d, center = c(3, 4), scale = c(2, 4))
+```
+
+## Data properties
+
+The workhorse function to get a comprehensive summary of data properties is `describe_distribution()`, which combines a set of indices (e.g., measures of centrality, dispersion, range, skewness, kurtosis, etc.) computed by other functions in `{datawizard}`.
+
+```{r}
+describe_distribution(mtcars$wt)
+```
```

Comment:
add an example to show that it also works with a full dataset?

Reply:
Done.

But the table output overflows in PDF; we will need to deal with that somehow.